docs/developer/developer-getting-started.md

```diff
@@ -0,0 +1,283 @@
+# Overview
+
+The intent of this document is to be a high level quick start guide to get up and running quickly. For a more in depth review on configuration please see [CONTRIBUTING.MD](../../CONTRIBUTING.md#running-the-ratify-cli).
+
+# Getting started
+
+High-level steps of this document:
+
+1. Start devcontainer
+1. Set aliases and environment variables
+1. Build images
+1. Create Kind cluster
+1. Install Gatekeeper (from repo via Helm)
+1. Install Ratify (from locally built image)
+1. Install a verifier
+1. Apply constraints and templates
+1. Validate
+
+## Devcontainer
+
+Start the devcontainer using the command pallet or the green button in the lower left corner of VSCode.
+
+## Create aliases
+
+```bash
+alias k="kubectl"
+```
+
+## Export variables
+
+```bash
+export RATIFY_NAMESPACE=ratify-service
+export KUBERNETES_VERSION=1.25.4
```

Comment:
Now that we are supporting 3.10 and 3.11 of GK. Do you think it might help to add a GK version variable too and then specify version for GK installation?